tests/system/test_requests.py

```diff
@@ -55,6 +54,11 @@ def test_validation_2_fail(self):
         assert r.status_code == 400, r.status_code
         assert "Problem validating JSON document against schema" in r.content, r.content
 
+    def test_frontend_disabled(self):
```

Comment:
I think this should make it clear that the default is "off". Could be achieved by just renaming test name to `test_frontend_default_disabled` or similar.